app/controllers/api/v1/users_controller.rb

```diff
@@ -62,7 +63,12 @@ def user_params
 			end
 
 			def set_user
-				@user = User.find( id: params[ :id ] )
+				@user = User.find( params[ :id ] )
+				ExceptionHandler.record_not_found if @user.blank?
+			end
+
+			def render_json(user)
```

Comment:
Move this method to a module and include module at the Application controller.

Reply:
> Move this method to a module and include module at the Application controller.

Done.
https://github.com/sas2job/test-rails-api/blob/feature/json_serializer/app/controllers/concerns/exception_handler.rb